src/zarr/core/metadata/v3.py

```diff
@@ -135,8 +135,8 @@ def parse_storage_transformers(data: object) -> tuple[dict[str, JSON], ...]:
 
 class V3JsonEncoder(json.JSONEncoder):
     def __init__(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
We should aim for something a bit more explicit than this function signature :sweat_smile: . I'm pretty sure a better signature here would have made this bug easier to catch. (not a blocker for this PR)